wks-recruiter-interface/src/app/layout/tests/tests-create/tests-create.component.html

```diff
@@ -24,9 +24,17 @@ <h4 class="modal-title">{{title}}</h4>
     </div>
     <div class="form-group">
       <label for="testLanguage">Language</label>
-      <input type="text" class="form-control input-underline input-lg" placeholder="Language" id="language" name="language" [(ngModel)]="test.language"
-        #language="ngModel" required minlength="3" pattern="^[a-z]+">
-      <div *ngIf="testCreationForm.submitted && language.invalid" class="alert alert-danger">
+        <select class="form-control" name="language" [(ngModel)]="test.language" #language="ngModel" required>
```

Comment:
Nie można tego wrzucić w tablicę lub mapę w komponencie? W html'u skondensowałoby się to do jednej linijki wtedy i wyglądałoby estetyczniej.